easy-ui-react/src/ForgeLayout/ForgeLayoutControls.module.scss

```diff
@@ -0,0 +1,75 @@
+@use "../styles/common" as *;
```

Comment:
**note to reviewer**: it is expected that this layout is not optimized for smaller screens